### ISSUE TITLE
ArrayIndexOutOfBoundsException thrown when using setUserEmails with EmailCryptType set to MD5 or SHA256

### ISSUE THREAD

**Issue by ibrahim-mubarak** [AppsFlyerSDK/appsflyer-flutter-plugin#174]
**Describe the bug**
ArrayIndexOutOfBoundsException thrown when using setUserEmails with EmailCryptType set to MD5 or SHA256

**To Reproduce**

Set  user email in the following way

```dart
instance.setUserEmails(
  [email],
  EmailCryptType.EmailCryptTypeSHA1,
);

```
1. When EmailCryptType.MD5 is set

```
E/MethodChannel#af-api( 6741): Failed to handle method call
E/MethodChannel#af-api( 6741): java.lang.ArrayIndexOutOfBoundsException: length=2; index=2
E/MethodChannel#af-api( 6741): 	at com.appsflyer.appsflyersdk.AppsflyerSdkPlugin.setUserEmailsWithCryptType(AppsflyerSdkPlugin.java:501)
E/MethodChannel#af-api( 6741): 	at com.appsflyer.appsflyersdk.AppsflyerSdkPlugin.onMethodCall(AppsflyerSdkPlugin.java:243)
```

2. When EmailCryptType.SHA256 is set
```
E/MethodChannel#af-api( 6741): Failed to handle method call
E/MethodChannel#af-api( 6741): java.lang.ArrayIndexOutOfBoundsException: length=2; index=3
E/MethodChannel#af-api( 6741): 	at com.appsflyer.appsflyersdk.AppsflyerSdkPlugin.setUserEmailsWithCryptType(AppsflyerSdkPlugin.java:501)
E/MethodChannel#af-api( 6741): 	at com.appsflyer.appsflyersdk.AppsflyerSdkPlugin.onMethodCall(AppsflyerSdkPlugin.java:243)
E/MethodChannel#af-api( 6741): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
```

**Expected behavior**
No error is thrown

**Smartphone (please complete the following information):**
 - Device: OnePlus 8T
 - OS:  Android 11

**Comment by GM-appsflyer** [comment]
hEY @ibrahim-mubarak ,

Thanks for reaching out!
SHA1 & MD5 were removed from our native SDK, that's what causing the issue.
A fix for this issue will be released in the next version (6.4.3).

**Comment by GM-appsflyer** [comment]
A fix was released in V6.4.4+1.
Only SHA256 is allowed for encryption.
